javatests/arcs/sdk/examples/testing/ComputePeopleStatsTest.kt

```diff
@@ -52,6 +52,7 @@ class ComputePeopleStatsTest {
                 harness.people.store(Person(30.0))
             )
         }.joinAll()
+        harness.people.getProxy().waitForIdle()
```

Comment:
Rather than exposing the proxy to clients, it'd introduce less [production-]api surface area to add a `waitForIdle()` (or maybe a `waitForStorageIdle()`, or something) to the harness instead.